README.md

```diff
@@ -2,14 +2,20 @@
 
 CDS APIs are concentrated into two areas: Candidate Data & Candidate Search.
 
-## Candidate Data APIs
+## Candidate Data
+### Uses
+* [Searching Private Candidates](/SearchingPrivateCandidates.md)
```

Comment:
I would move this under Candidate Search instead

Reply:
I left it there because its specific to using candidate search for Candidate Data aka mycandidates